index.html

```diff
@@ -204,5 +204,21 @@ <h4 class="connect">LET&#39;S CONNECT</h4>
         </li>
       </ul>
     </section>
+
+    <section class="contact-form">
+      <h2 class="contact-form-heading">Contact me</h2>
+      <p class="contact-form-description">If you have an application you are interested in developing, a feature that 
+          you need built or a project that needs coding. I&#39;d love to help with it </p>
+      <form action="https://formspree.io/f/mpznwdqe" class="contact-form-content" method="post">
+          <input type="text" name="username" class="contact-form-username"
+            placeholder ="enter username" required="required" maxlength="30" >
+          <input type="email" name="email" class=" contact-form-email"
+            placeholder="enter email" required="required">
+      <textarea name="message" cols="30" rows="10" class="contact-form-message message"
```

Comment:
- [ ] You did a wonderful job using <textarea> here :+1: :100:, but as a project requirement all fields should have the correct associated placeholder text.

**Figma template**

![Screenshot from 2023-01-18 05-26-17](https://user-images.githubusercontent.com/105117832/213147979-059d6696-afa9-428f-adb2-d7c863e13b28.png)

**Your design**

![Screenshot from 2023-01-18 05-28-38](https://user-images.githubusercontent.com/105117832/213148176-e042994f-3121-47a7-8342-842ae5ce38e1.png)